source/optional_features/custom_metadata/index.html.md.erb

```diff
@@ -1,15 +1,17 @@
 ---
 title: Adding custom metadata
 weight: 135
-last_reviewed_on: 2020-04-21
+last_reviewed_on: 2020-05-05
 review_in: 2 months
 ---
 
 # Adding custom metadata
 
 You can add custom metadata to a new payment. For example, you can add a reference number from your finance or accounting system, so you can reconcile the payment later.
 
-You add metadata when you make an API call to create a new payment. You cannot add metadata to [Direct Debit transactions](/direct_debit/#direct-debit) or [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/).
+You add metadata when you make an API call to create a new payment. You cannot add metadata to [Direct Debit transactions](/direct_debit/#direct-debit).
+
+If you're using [payment links](https://www.payments.service.gov.uk/govuk-payment-pages/), you can add reporting columns instead by signing in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/login).
```

Comment:
To avoid 'instead', how about "you can add metadata by adding reporting columns in the [GOV.UK Pay admin tool]"?

It looks like the cost centre code example is covered by the first para of the page: "For example, you can add a reference number from your finance or accounting system, so you can reconcile the payment later."

Reply:
yep sounds good to me!